independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/headers/NewCookieHeaderDelegate.java

```diff
@@ -71,7 +74,8 @@ public Object fromString(String newCookie) throws IllegalArgumentException {
             cookieValue = "";
         }
 
-        return new NewCookie(cookieName, cookieValue, path, domain, version, comment, maxAge, expiry, secure, httpOnly);
+        return new NewCookie(cookieName, cookieValue, path, domain, version, comment, maxAge, expiry, secure, httpOnly,
+                sameSite);
```

Comment:
```suggestion
        return new NewCookie.Builder(cookieName)
                .value(cookieValue)
                .path(path)
                .domain(domain)
                .version(version)
                .maxAge(maxAge)
                .expiry(expiry)
                .secure(secure)
                .httpOnly(httpOnly)
                .sameSite(sameSite)
                .build()
```

Reply:
I propose you push this as a separate commit 😉